catalyst_voices/packages/catalyst_voices_models/lib/src/brands.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_assets/generated/colors.gen.dart';
+
+import 'package:flutter/material.dart';
+
+// [brands] is a Map that stores the ThemeData for all the
+// available brands in the Catalyst Voices app.
+// This map is used by the [BrandBloc] to properly
+// switch the active theme based on the BlocState.
+
+final Map<BrandKey, ThemeData> brands = {
```

Comment:
Themes are usually objects that create a lot of objects themselves. Do you think we could create a builder to return ThemeData by the BrandKey? It can be also delayed until later since initially it won't be too problematic.